### PR TITLE
fix release version and _index

### PR DESCRIPTION
**Pull request by sunzhaoyang**
### First-time contributors' checklist 

- [x] I've signed [**Contributor License Agreement**](https://cla-assistant.io/pingcap/docs-dm) that's required for repo owners to accept my contribution.

### Which DM version(s) do your changes apply to? (Required)

- [x] master (the latest development version)
- [x] v5.3 (TiDB DM 5.3 versions)
- [ ] v2.0 (TiDB DM 2.0 versions)
- [ ] v1.0 (TiDB DM 1.0 versions)

### What is the related PR or file link(s)?

- This PR is translated from:
- Other reference link(s):

### Do your changes match any of the following descriptions?

- [ ] Delete files
- [ ] Change aliases
- [ ] Need modification after applied to another branch 
- [ ] Might cause conflicts after applied to another branch

### CODE REVIEW

**Comment by Liuxiaozhen12** [comment]
The release of DM 5.3 needs to update some descriptions about v2.0.X in the code examples and to test the updated document. Could you help to review this PR? /cc @lance6716 @GMHDBJD @lichunzhu

File: en/faq.md [hunk 1]
```diff
@@ -126,9 +126,9 @@ Set the parameters below to a value larger than the default 67108864 (64M).
 
 For details, see [Loader solution](https://docs.pingcap.com/tidb/stable/loader-overview#solution).
 
-## How to handle the error `Error 1054: Unknown column 'binlog_gtid' in 'field list'` that occurs when existing DM migration tasks of an DM 1.0 cluster are running on a DM 2.0 cluster?
+## How to handle the error `Error 1054: Unknown column 'binlog_gtid' in 'field list'` that occurs when existing DM migration tasks of an DM 1.0 cluster are running on a DM 2.0 newer cluster?
```
Comment (lance6716) [inline en/faq.md:129]:
```suggestion
## How to handle the error `Error 1054: Unknown column 'binlog_gtid' in 'field list'` that occurs when existing DM migration tasks of an DM 1.0 cluster are running on a DM 2.0 or newer cluster?
```

**Comment by lance6716** [comment]
rest lgtm

File: zh/maintain-dm-using-tiup.md [hunk 2]
```diff
@@ -267,38 +267,6 @@ tiup dm patch prod-cluster /tmp/dm-master-hotfix.tar.gz -R dm-master
 tiup dm patch prod-cluster /tmp/dm--hotfix.tar.gz -N 172.16.4.5:8261
 ```
 
-## 导入 DM-Ansible 部署的 DM 1.0 集群并升级
```
Comment (Liuxiaozhen12) [inline zh/maintain-dm-using-tiup.md:270]: @lance6716 Please help to check whether we should delete or reserve this section, thanks!

File: en/TOC.md [hunk 3]
```diff
@@ -45,8 +45,7 @@
     - [Maintain DM Clusters Using dmctl](dmctl-introduction.md)
     - [Maintain DM Clusters Using OpenAPI](open-api.md)
   + Cluster Upgrade
-    - [Manually Upgrade from v1.0.x to v2.0.x](manually-upgrade-dm-1.0-to-2.0.md)
-    - [Upgrade Between v1.0.x](upgrade-dm-1.0.md)
+    - [Manually Upgrade from v1.0.x to v2.0.x+](manually-upgrade-dm-1.0-to-2.0.md)
```
Comment (glorv) [inline en/TOC.md:48]: `v2.0.x+` should be replaced with `v2.0+` or `v2.0.0+`

Comment (sunzhaoyang) [inline en/TOC.md:48]:
```suggestion
    - [Manually Upgrade from v1.0.x to v2.0+](manually-upgrade-dm-1.0-to-2.0.md)
```

File: zh/TOC.md [hunk 4]
```diff
@@ -45,8 +45,7 @@
     - [使用 dmctl 运维集群](dmctl-introduction.md)
     - [使用 OpenAPI 运维集群](open-api.md)
   + 升级版本
-    - [1.0.x 到 2.0.x 手动升级](manually-upgrade-dm-1.0-to-2.0.md)
-    - [1.0.x 版本间升级](upgrade-dm-1.0.md)
+    - [1.0.x 到 2.0.x+ 手动升级](manually-upgrade-dm-1.0-to-2.0.md)
```
Comment (sunzhaoyang) [inline zh/TOC.md:48]:
```suggestion
    - [1.0.x 到 2.0+ 手动升级](manually-upgrade-dm-1.0-to-2.0.md)
```

File: zh/manually-upgrade-dm-1.0-to-2.0.md [hunk 5]
```diff
@@ -119,7 +117,7 @@ from:
 
 ## 第 4 步：升级数据迁移任务
 
-1. 使用 [`operate-source`](manage-source.md#数据源操作) 命令将 [准备 v2.0.x 的配置文件](#第-1-步准备-v20x-的配置文件) 中得到的上游数据库 source 配置加载到 v2.0.x 集群中。
+1. 使用 [`operate-source`](manage-source.md#数据源操作) 命令将 [准备 v2.0.x+ 的配置文件](#第-1-步准备-v20x-的配置文件) 中得到的上游数据库 source 配置加载到 v2.0.x+ 集群中。
```
Comment (sunzhaoyang) [inline zh/manually-upgrade-dm-1.0-to-2.0.md:120]:
```suggestion
1. 使用 [`operate-source`](manage-source.md#数据源操作) 命令将 [准备 v2.0+ 的配置文件](#第-1-步准备-v20x-的配置文件) 中得到的上游数据库 source 配置加载到 v2.0+ 集群中。
```

File: zh/manually-upgrade-dm-1.0-to-2.0.md [hunk 6]
```diff
@@ -101,15 +99,15 @@ from:
...
-## 第 2 步：部署 v2.0.x 集群
+## 第 2 步：部署 v2.0.x+ 集群
 
 > **注意：**
 >
-> 如果已有其他可用的 v2.0.x 集群，可跳过此步。
+> 如果已有其他可用的 v2.0.x+ 集群，可跳过此步。
 
-[使用 TiUP](deploy-a-dm-cluster-using-tiup.md) 按所需要节点数部署新的 v2.0.x 集群。
+[使用 TiUP](deploy-a-dm-cluster-using-tiup.md) 按所需要节点数部署新的 v2.0.x+ 集群。
```
Comment (sunzhaoyang) [inline zh/manually-upgrade-dm-1.0-to-2.0.md:110]:
```suggestion
[使用 TiUP](deploy-a-dm-cluster-using-tiup.md) 按所需要节点数部署新的 v2.0+ 集群。
```

File: zh/manually-upgrade-dm-1.0-to-2.0.md [hunk 7]
```diff
@@ -136,7 +134,7 @@ from:
         +------------------+-------------------------+------------+
         ```
 
-3. 更新 v1.0.x 的数据迁移任务配置文件以启动新的 v2.0.x 数据迁移任务。
+3. 更新 v1.0.x 的数据迁移任务配置文件以启动新的 v2.0.x+ 数据迁移任务。
```
Comment (sunzhaoyang) [inline zh/manually-upgrade-dm-1.0-to-2.0.md:137]:
```suggestion
3. 更新 v1.0.x 的数据迁移任务配置文件以启动新的 v2.0+ 数据迁移任务。
```

File: zh/manually-upgrade-dm-1.0-to-2.0.md [hunk 8]
```diff
@@ -161,8 +159,8 @@ from:
             >
             > 如在 source 配置中启动了 `enable-gtid`，当前需要通过解析 binlog 或 relay log 文件获取 binlog position 对应的 GTID sets 并在 `meta` 中设置为 `binlog-gtid`。
 
-4. 使用 [`start-task`](create-task.md) 命令以 v2.0.x 的数据迁移任务配置文件启动升级后的数据迁移任务。
+4. 使用 [`start-task`](create-task.md) 命令以 v2.0.x+ 的数据迁移任务配置文件启动升级后的数据迁移任务。
```
Comment (sunzhaoyang) [inline zh/manually-upgrade-dm-1.0-to-2.0.md:162]:
```suggestion
4. 使用 [`start-task`](create-task.md) 命令以 v2.0+ 的数据迁移任务配置文件启动升级后的数据迁移任务。
```

File: zh/manually-upgrade-dm-1.0-to-2.0.md [hunk 9]
```diff
@@ -161,8 +159,8 @@ from:
...
 
 5. 使用 [`query-status`](query-status.md) 命令确认数据迁移任务是否运行正常。
 
-如果数据迁移任务运行正常，则表明 DM 升级到 v2.0.x 的操作成功。
+如果数据迁移任务运行正常，则表明 DM 升级到 v2.0.x+ 的操作成功。
```
Comment (sunzhaoyang) [inline zh/manually-upgrade-dm-1.0-to-2.0.md:166]:
```suggestion
如果数据迁移任务运行正常，则表明 DM 升级到 v2.0+ 的操作成功。
```

File: en/manually-upgrade-dm-1.0-to-2.0.md [hunk 10]
```diff
@@ -1,35 +1,35 @@
 ---
-title: Manually Upgrade TiDB Data Migration from v1.0.x to v2.0.x
-summary: Learn how to manually upgrade TiDB data migration from v1.0.x to v2.0.x.
+title: Manually Upgrade TiDB Data Migration from v1.0.x to v2.0.x+
+summary: Learn how to manually upgrade TiDB data migration from v1.0.x to v2.0.x+.
 ---
 
-# Manually Upgrade TiDB Data Migration from v1.0.x to v2.0.x
+# Manually Upgrade TiDB Data Migration from v1.0.x to v2.0.x+
```
Comment (sunzhaoyang) [inline en/manually-upgrade-dm-1.0-to-2.0.md:6]:
```suggestion
# Manually Upgrade TiDB Data Migration from v1.0.x to v2.0+
```

File: en/manually-upgrade-dm-1.0-to-2.0.md [hunk 11]
```diff
@@ -1,35 +1,35 @@
 ---
-title: Manually Upgrade TiDB Data Migration from v1.0.x to v2.0.x
-summary: Learn how to manually upgrade TiDB data migration from v1.0.x to v2.0.x.
+title: Manually Upgrade TiDB Data Migration from v1.0.x to v2.0.x+
```
Comment (sunzhaoyang) [inline en/manually-upgrade-dm-1.0-to-2.0.md:2]:
```suggestion
title: Manually Upgrade TiDB Data Migration from v1.0.x to v2.0+
```

File: en/manually-upgrade-dm-1.0-to-2.0.md (hunk 10 above)
Comment (sunzhaoyang) [inline en/manually-upgrade-dm-1.0-to-2.0.md:6]:
```suggestion
# Manually Upgrade TiDB Data Migration from v1.0.x to v2.0+
```

File: en/manually-upgrade-dm-1.0-to-2.0.md [hunk 12]
```diff
@@ -1,35 +1,35 @@
...
+summary: Learn how to manually upgrade TiDB data migration from v1.0.x to v2.0.x+.
 ---
 
-# Manually Upgrade TiDB Data Migration from v1.0.x to v2.0.x
+# Manually Upgrade TiDB Data Migration from v1.0.x to v2.0.x+
 
-This document introduces how to manually upgrade the TiDB DM tool from v1.0.x to v2.0.x. The main idea is to use the global checkpoint information in v1.0.x to start a new data migration task in the v2.0.x cluster.
+This document introduces how to manually upgrade the TiDB DM tool from v1.0.x to v2.0.x+. The main idea is to use the global checkpoint information in v1.0.x to start a new data migration task in the v2.0.x+ cluster.
```
Comment (sunzhaoyang) [inline en/manually-upgrade-dm-1.0-to-2.0.md:8]:
```suggestion
This document introduces how to manually upgrade the TiDB DM tool from v1.0.x to v2.0+. The main idea is to use the global checkpoint information in v1.0.x to start a new data migration task in the v2.0+ cluster.
```

File: en/manually-upgrade-dm-1.0-to-2.0.md [hunk 13]
```diff
@@ -1,35 +1,35 @@
...
 
-For how to automatically upgrade the TiDB DM tool from v1.0.x to v2.0.x, refer to [Using TiUP to automatically import the 1.0 cluster deployed by DM-Ansible](maintain-dm-using-tiup.md#import-and-upgrade-a-dm-10-cluster-deployed-using-dm-ansible).
+For how to automatically upgrade the TiDB DM tool from v1.0.x to v2.0.x+, refer to [Using TiUP to automatically import the 1.0 cluster deployed by DM-Ansible](maintain-dm-using-tiup.md#import-and-upgrade-a-dm-10-cluster-deployed-using-dm-ansible).
 
 > **Note:**
 >
-> - Currently, upgrading DM from v1.0.x to v2.0.x is not supported when the data migration task is in the process of full export or full import.
+> - Currently, upgrading DM from v1.0.x to v2.0.x+ is not supported when the data migration task is in the process of full export or full import.
```
Comment (sunzhaoyang) [inline en/manually-upgrade-dm-1.0-to-2.0.md:14]:
```suggestion
> - Currently, upgrading DM from v1.0.x to v2.0+ is not supported when the data migration task is in the process of full export or full import.
```

File: en/manually-upgrade-dm-1.0-to-2.0.md [hunk 14]
```diff
@@ -1,35 +1,35 @@
...
 > - As the gRPC protocol used for interaction between the components of the DM cluster is updated greatly, you need to make sure that the DM components (including dmctl) use the same version before and after the upgrade.
-> - Because the metadata storage of the DM cluster (such as checkpoint, shard DDL lock status and online DDL metadata, etc.) is updated greatly, the metadata of v1.0.x cannot be reused automatically in v2.0.x. So you need to make sure the following requirements are satisfied before performing the upgrade operation:
+> - Because the metadata storage of the DM cluster (such as checkpoint, shard DDL lock status and online DDL metadata, etc.) is updated greatly, the metadata of v1.0.x cannot be reused automatically in v2.0.x+. So you need to make sure the following requirements are satisfied before performing the upgrade operation:
 >     - All data migration tasks are not in the process of shard DDL coordination.
 >     - All data migration tasks are not in the process of online DDL coordination.
 
 The steps for manual upgrade are as follows.
 
-## Step 1: Prepare v2.0.x configuration file
+## Step 1: Prepare v2.0.x+ configuration file
```
Comment (sunzhaoyang) [inline en/manually-upgrade-dm-1.0-to-2.0.md:22]:
```suggestion
## Step 1: Prepare v2.0+ configuration file
```

File: en/manually-upgrade-dm-1.0-to-2.0.md [hunk 15]
```diff
@@ -1,35 +1,35 @@
...
 
-The prepared configuration files of v2.0.x include the configuration files of the upstream database and the configuration files of the data migration task.
+The prepared configuration files of v2.0.x+ include the configuration files of the upstream database and the configuration files of the data migration task.
```
Comment (sunzhaoyang) [inline en/manually-upgrade-dm-1.0-to-2.0.md:24]:
```suggestion
The prepared configuration files of v2.0+ include the configuration files of the upstream database and the configuration files of the data migration task.
```

File: en/manually-upgrade-dm-1.0-to-2.0.md [hunk 16]
```diff
@@ -1,35 +1,35 @@
...
 
 ### Upstream database configuration file
 
-In v2.0.x, the [upstream database configuration file](source-configuration-file.md) is separated from the process configuration of the DM-worker, so you need to obtain the source configuration based on the [v1.0.x DM-worker configuration](https://docs.pingcap.com/tidb-data-migration/stable/dm-worker-configuration-file).
+In v2.0.x+, the [upstream database configuration file](source-configuration-file.md) is separated from the process configuration of the DM-worker, so you need to obtain the source configuration based on the [v1.0.x DM-worker configuration](https://docs.pingcap.com/tidb-data-migration/stable/dm-worker-configuration-file).
```
Comment (sunzhaoyang) [inline en/manually-upgrade-dm-1.0-to-2.0.md:28]:
```suggestion
In v2.0+, the [upstream database configuration file](source-configuration-file.md) is separated from the process configuration of the DM-worker, so you need to obtain the source configuration based on the [v1.0.x DM-worker configuration](https://docs.pingcap.com/tidb-data-migration/stable/dm-worker-configuration-file).
```

File: en/TOC.md (hunk 3 above)
Comment (sunzhaoyang) [inline en/TOC.md]:
```suggestion
    - [Manually Upgrade from v1.0.x to v2.0+](manually-upgrade-dm-1.0-to-2.0.md)
```

Comment (sunzhaoyang) [inline en/TOC.md]:
```suggestion
    - [Manually Upgrade from v1.0.x to v2.0+](manually-upgrade-dm-1.0-to-2.0.md)
```

File: en/maintain-dm-using-tiup.md [hunk 17]
```diff
@@ -273,7 +273,7 @@ tiup dm patch prod-cluster /tmp/dm--hotfix.tar.gz -N 172.16.4.5:8261
 > - You need to stop the original cluster before importing.
 > - Don't run `stop-task` for tasks that need to be upgraded to 2.0.
 > - TiUP only supports importing to a DM cluster of v2.0.0-rc.2 or a later version.
-> - The `import` command is used to import data from a DM 1.0 cluster to a new DM 2.0 cluster. If you need to import DM migration tasks to an existing DM 2.0 cluster, refer to [Manually Upgrade TiDB Data Migration from v1.0.x to v2.0.x](manually-upgrade-dm-1.0-to-2.0.md).
+> - The `import` command is used to import data from a DM 1.0 cluster to a new DM 2.0 cluster. If you need to import DM migration tasks to an existing DM 2.0 cluster, refer to [Manually Upgrade TiDB Data Migration from v1.0.x to v2.0.x+](manually-upgrade-dm-1.0-to-2.0.md).
```
Comment (sunzhaoyang) [inline en/maintain-dm-using-tiup.md:276]:
```suggestion
> - The `import` command is used to import data from a DM 1.0 cluster to a new DM 2.0 cluster. If you need to import DM migration tasks to an existing DM 2.0 cluster, refer to [Manually Upgrade TiDB Data Migration from v1.0.x to v2.0+](manually-upgrade-dm-1.0-to-2.0.md).
```

File: en/TOC.md (hunk 3 above)
Comment (sunzhaoyang) [inline en/TOC.md]:
```suggestion
    - [Manually Upgrade from v1.0.x to v2.0+](manually-upgrade-dm-1.0-to-2.0.md)
```

File: en/manually-upgrade-dm-1.0-to-2.0.md [hunk 18]
```diff
@@ -1,35 +1,35 @@
...
 
 > **Note:**
 >
-> - Currently, upgrading DM from v1.0.x to v2.0.x is not supported when the data migration task is in the process of full export or full import.
+> - Currently, upgrading DM from v1.0.x to v2.0+ is not supported when the data migration task is in the process of full export or full import.
 > - As the gRPC protocol used for interaction between the components of the DM cluster is updated greatly, you need to make sure that the DM components (including dmctl) use the same version before and after the upgrade.
-> - Because the metadata storage of the DM cluster (such as checkpoint, shard DDL lock status and online DDL metadata, etc.) is updated greatly, the metadata of v1.0.x cannot be reused automatically in v2.0.x. So you need to make sure the following requirements are satisfied before performing the upgrade operation:
+> - Because the metadata storage of the DM cluster (such as checkpoint, shard DDL lock status and online DDL metadata, etc.) is updated greatly, the metadata of v1.0.x cannot be reused automatically in v2.0.x+. So you need to make sure the following requirements are satisfied before performing the upgrade operation:
```
Comment (lance6716) [inline en/manually-upgrade-dm-1.0-to-2.0.md:16]: we can search and replace `v2.0.x+` to `v2.0+`

**Review by lance6716 — COMMENTED** [review]
rest lgtm

File: zh/_index.md [hunk 19]
```diff
@@ -6,17 +6,17 @@ aliases: ['/docs-cn/tidb-data-migration/dev/']
...
-- 与上下游数据库及 DM 各组件间连接的 [TLS 支持](enable-tls.md)。
-- 实验性地支持从 MySQL 8.0 迁移数据。
+- 合并单行数据的多次变更，将点查更新合并为批量操作，实现以更低的延迟将数据从 MySQL 同步数据到 TiDB。
+- 增加 DM [OpenAPI](open-api.md) 以更方便地管理集群（实验特性）。
 
-目前 DM 工具没有图形化管理界面，对大量数据迁移任务管理的支持有限。**如果您需要操作大量数据迁移任务，请谨慎选择使用 DM。**
+**重要说明**
+
+- 由于某些原因，DM 的 GitHub 仓库已移到 [https://github.com/pingcap/ticdc/tree/master/dm](https://github.com/pingcap/ticdc/tree/master/dm)。请到新的仓库提交问题，以获得后续反馈。
+- DM 新版本的发布时间将与 [TiCDC](https://docs.pingcap.com/zh/tidb/stable/ticdc-overview) 保持一致，因此版本号也将保持一致。自 v2.1.0 起，DM 版本号将改为 v5.3.0。
```
Comment (Liuxiaozhen12) [inline zh/_index.md:19]:
```suggestion
- 截至 DM v2.0，DM 采用独立于 TiDB 的版本号。DM v2.0 后，DM 采用与 TiDB 相同的版本号。因此，DM v2.0 的后一版本为 DM v5.3。DM v2.0 到 v5.3 无兼容性变更，升级过程与正常升级无差异，仅仅是版本号上的提升。
```

**Review by Liuxiaozhen12 — APPROVED** [review]

**Pull request merged**